extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -10,7 +10,17 @@ define('ext.wikia.adEngine.video.googleIma', [
 		isAdsManagerLoaded = false,
 		videoMock = document.createElement('video');
 
+	function isInitialized() {
+		return !!window.google && !!window.google.ima;
+	}
+
 	function init() {
+		if (isInitialized()) {
+			return new Promise(function (resolve, reject) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "reject". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)